### PR TITLE
Text utility

### PR DESCRIPTION
**Pull request by sahiba0915**
Hello team , please review my code.

Live preview: https://text-utility-cl.netlify.app/component/text/text-utility.html

### CODE REVIEW

File: component/text/text.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  font-family: var(--primary-font);
+}
+:root {
+  --primary-text: #000000;
+  --secondary-text: #747171;
+  --primary-color: #f9a826;
+  --secondary-color: #ffffff;
+  --btn-color: #965c49;
+  --box-shadow: #979494;
+  --primary-font: "Poppins", sans-serif;
```
Comment (akash-1318) [inline component/text/text.css:14]: For better readability keep same types of variables together.

**Review by akash-1318 — COMMENTED** [review]
Added a simple suggestion other than that looks good.

**Review by chanchal-panpaliya — COMMENTED** [review]
The code is good and well structured.

File: component/text/text.css (hunk 1 above)
Comment (sahiba0915) [reply component/text/text.css:14]: ok noted thanks

**Pull request merged**